README.md

```diff
@@ -49,9 +49,10 @@ In your project's Gruntfile, add a section named `not_constantinople` to the dat
 ```js
 grunt.initConfig({
   not_constantinople: {
-    istanbul_options{
+    customized_coverage: {
```

Comment:
I wanted to make this task name more clear as I was unsure at first if this was a special name for Istanbul-specific options used by the multitask, or if it was a task name itself (which it was).

Additionally, there was a missing `: ` thus causing a syntax error.